tests/openbis_functions/test_interbis.py

```diff
@@ -145,9 +191,9 @@ def setup(pytestconfig):
     o.logout()
 
 
-@pytest.mark.login
-@pytest.mark.parametrize("write, sheet_name, path", [(False, "metadata", ""),
-                                                     (True, "some_named_sheet", Filepaths.excel_output.value)])
+@ pytest.mark.login
```

Comment:
why did you remove the space?

Reply:
I have no idea what happend in those decorators, must have been a formatter error. I havent added random spaces by hand. I have removed the spaces.